users/vieting/experiments/librispeech/librispeech_100_ctc/fairseq_finetuning/ctc_standalone/sisyphus_configs/config_main_pretrain_finetune.py

```diff
@@ -0,0 +1,22 @@
+from i6_experiments.users.vieting.experiments.librispeech.\
+    librispeech_100_ctc.fairseq_finetuning.ctc_standalone.experiments.ctc_phon.baseline import eow_phon_ls100_ctc_base
```

Comment:
Line too long? You can also apply `black` to your files which will take care of this automatically.